packages/portal-proto/src/features/featureFlags.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ *  Feature flags for GDC Portal.
+ *  Define a flag in either local storage or  environment variable
+ *  Note that in a production build NextJS will compile in the value of the ENV
+ *  variable when it is built
+ *
+ */
+let localStorage: Storage = {} as Storage;
+if (typeof window !== "undefined") localStorage = window.localStorage;
+
+/**
+ * Feature flag for Single Cell RNA-seq
+ */
+export const DISPLAY_SC_RNA_SEQ_APP =
+  localStorage.GDC_PORTAL_DISPLAY_SC_RNA_SEQ_APP ||
+  process.env.NEXT_PUBLIC_GDC_PORTAL_DISPLAY_SC_RNA_SEQ_APP ||
+  false;
```

Comment:
Would optional chaining be appropriate here?

```suggestion
export const DISPLAY_SC_RNA_SEQ_APP =
  localStorage?.GDC_PORTAL_DISPLAY_SC_RNA_SEQ_APP ||
  process?.env?.NEXT_PUBLIC_GDC_PORTAL_DISPLAY_SC_RNA_SEQ_APP ||
  false;
```

Reply:
I agree that would be safer.